packages/fxa-settings/src/pages/PostVerify/ThirdPartyAuthCallback/index.tsx

```diff
@@ -129,28 +103,51 @@ const ThirdPartyAuthCallback = ({
     return { code, provider };
   };
 
+  const navigateNext = async (linkedAccount: LinkedAccountData) => {
+    if (!integration) {
+      return;
+    }
+
+    const navigationOptions = {
+      email: linkedAccount.email,
+      signinData: {
+        uid: linkedAccount.uid,
+        sessionToken: linkedAccount.sessionToken,
+        verified: true,
+      },
+      integration,
+      finishOAuthFlowHandler,
+      queryParams: location.search,
+    };
+
+    const { error: navError } = await handleNavigation(navigationOptions, {
+      handleFxaLogin: false,
+      handleFxaOAuthLogin: true,
```

Comment:
I think we'll want both of these to be `false` since if the user signs in through the Sync flow here they'll be taken to `set_password` where we'll need to send these up after they finish that page. Related: https://github.com/mozilla/fxa/pull/18044/files#diff-b3b388fc2f5d3eeb77ab09e9410852e737adee6fb1fc9649ae8367627abc16dfR92)